src/cascadia/WindowsTerminal/main.cpp

```diff
@@ -4,14 +4,50 @@
 #include "pch.h"
 #include "AppHost.h"
 
+#include <wil/result.h>
+
 using namespace winrt;
 using namespace Windows::UI;
 using namespace Windows::UI::Composition;
 using namespace Windows::UI::Xaml::Hosting;
 using namespace Windows::Foundation::Numerics;
 
+static constexpr std::wstring_view ImageArchitectureToString(USHORT imageArchitecture)
+{
+    // clang-format off
+    return imageArchitecture == IMAGE_FILE_MACHINE_I386 ? L"i386" :
+                                IMAGE_FILE_MACHINE_AMD64 ? L"AMD64" :
+                                IMAGE_FILE_MACHINE_ARM64 ? L"ARM64" :
+                                IMAGE_FILE_MACHINE_ARM ? L"ARM" :
+                                L"Unknown";

```

Comment:
nnnng. I hate to say this. But doesn't L"Unknown" need to go into a resource file for localization?